src/api.js

```diff
@@ -0,0 +1,86 @@
+let debug = require('debug')('app:main');
+let APIBuilder = require('taskcluster-lib-api');
+let taskcluster = require('taskcluster-client');
+let uuid = require('uuid');
+
+let builder = new APIBuilder({
+  title:    'AMQP Messages API Documentation',
+  description: [
+    'This service is responsible for making pulse messages accessible',
+    'from browsers and cli. There are API endpoints to',
+    'bind / unbind to an exchange and pause / resume listening from a queue',
+  ].join('\n'),
+  serviceName: 'tcevents',
+  version: 'v1',
+  errorCodes: {
+    InputValidation:  400, // For JSON schema errors (bad exchange).
```

Comment:
This is declared in:
https://github.com/taskcluster/taskcluster-lib-api/blob/master/src/errors.js#L8